scss/control.scss

```diff
@@ -147,14 +147,22 @@
 		outline: none;
 		// padding: 0;
 		line-height: 14px;  /* For IE 8 compatibility */
-		padding: (($select-input-internal-height - 14) / 2 - 2) 0 (($select-input-internal-height - 14) / 2 + 2);  /* For IE 8 compatibility */
+		padding: -1 (($select-input-internal-height - 14) / 2 - 1) 0 (($select-input-internal-height - 14) / 2);  /* For IE 8 compatibility */
 		-webkit-appearance: none;
 
 		.is-focused & {
 			cursor: text;
 		}
 	}
+}
 
+// in FF, the input field jumps a pixel when enabled, this adjust for that so the label and input are perfectly aligned
```

Comment:
Using vendor specific hacks should not be used. If this selector ever goes away then the original issue may reappear. To ensure they are perfectly aligned I think we should spend a bit more time digging into why they aren't instead of adding a band aid :smile:

Reply:
@agirton I don't disagree with you, but: https://giphy.com/gifs/13XW2MJE0XCoM0

I tried to find a fix that's not vendor specific before your comment but didn't get anywhere. I would welcome any assist from the community. :D